### PR TITLE
feat: sac and clubs info page

### LINKED ISSUE #271: feat(guide): create sac and clubs info page
The design is available on [figma](https://www.figma.com/file/VtzXViXjXah3IuWn2xBRae/Website?node-id=736%3A51).
157028391-3645bfc2-0065-42bb-b3ab-dffa65ac75de

### PR DESCRIPTION
**Pull request by anish-patro**
### Your checklist for this pull request

🚨Please review the [guidelines for contributing](https://github.com/Monday-Morning/project-tahiti/blob/main/CONTRIBUTING.md) to this repository.

- [x] Make sure you are requesting to **pull a topic/feature/bugfix branch** (right side). Don't request your default branch!
- [x] Make sure you are making a pull request against the **default branch** (left side). Also you should start _your branch_ off _default branch_.
- [x] Check the commit's or even all commits' message styles matches our requested structure.
- [x] Check your code additions will fail neither code linting checks nor unit test.
- [x] I have added necessary documentation (if appropriate)

### Linked Issues 

Fixes #271 

### Description

Screenshot (34)
please describe your pull request.

❤️Thank you!

### Post merge checklist

- [x] Follow steps from the [guidelines for contributing](https://github.com/Monday-Morning/project-tahiti/blob/main/CONTRIBUTING.md) to this repository.
- [x] If you are a new contributor, ping in the thread and one of the maintainers will add you to all-contributors list.

### CODE REVIEW

**Comment by 120EE0692** [comment]
@Anish-Sarawgi please change title of the PR and rebase with the main branch.

File: client/src/components/widgets/ClubCards.js [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+    padding: '5px 0px',
+    display: 'flex',
+    alignItems: 'center',
+    justifyContent: 'flex-start',
+    flexDirection: 'column',
+    [theme.breakpoints.up('sm')]: {
+      flexDirection: 'row',
+      justifyContent: 'center',
+      alignItems: 'center',
+      //  padding:'3px'
```
Comment (120EE0692) [inline client/src/components/widgets/ClubCards.js:177]: @anish-patro remove unnecessary comments.

File: client/src/components/widgets/ClubInfoNavbar.js [hunk 2]
```diff
@@ -0,0 +1,201 @@
...
+  //           {STAGE[key]}
+  //         </Typography>
+  //       ))}
+
+  //       <Container maxWidth='xm' sx={{ mt: 5 }}>
+  //         <div>{renderTeam()}</div>
+  //       </Container>
+  //     </div>
+
+  //   </>
```
Comment (120EE0692) [inline client/src/components/widgets/ClubInfoNavbar.js:70]: @anish-patro why this comment is there?

File: client/src/components/widgets/ClubInfoNavbar.js [hunk 3]
```diff
@@ -0,0 +1,201 @@
...
+//     className={classes.subCategory}
+//     onClick={() => {
+//       setStage(STAGE[key]);
+//     }}
+//   />
+// ))}
+// <Container maxWidth='xm' sx={{ mt: 5 }}>
+//   <div>{renderTeam()}</div>
+// </Container>
+// </div> }
```
Comment (120EE0692) [inline client/src/components/widgets/ClubInfoNavbar.js:201]: @anish-patro why this comment is there?

File: client/src/components/widgets/PostHolder.js [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+      direction='column'
+      justifyContent='center'
+      alignItems='flex-start'
+    >
+      <Typography variant='body2' className={classes.position}>
+        Second year
+      </Typography>
+      <Typography
+        variant='body2'
+        style={{ marginBottom: '5px' }}
```
Comment (120EE0692) [inline client/src/components/widgets/PostHolder.js:22]: @anish-patro no inline styles.

File: client/src/components/widgets/PostHolder.js [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+        className={classes.holderDetails}
+      >
+        name: Anish Patro
+      </Typography>
+      <Typography
+        variant='body2'
+        style={{ marginBottom: '5px' }}
```
Comment (120EE0692) [inline client/src/components/widgets/PostHolder.js:29]: @anish-patro no inline styles

File: client/src/components/widgets/PostHolder.js [hunk 6]
```diff
@@ -0,0 +1,71 @@
...
+        className={classes.holderDetails}
+      >
+        Roll: 121CE0877
+      </Typography>
+      <div className={classes.phoneDetails}>
+        <PhoneOutlined className={classes.phone} />
+        <Typography style={{ marginLeft: '10px' }} variant='body2'>
```
Comment (120EE0692) [inline client/src/components/widgets/PostHolder.js:36]: @anish-patro no inline styles.

File: client/src/components/widgets/SacInfoNavbar.js [hunk 7]
```diff
@@ -0,0 +1,189 @@
...
+// import { ButtonBase } from '@mui/material';
+
+const STAGE = {
+  SOB: 'SAC Office Bearers',
+  DNF: 'Dean’s Nominee for Fests',
+  DNS: 'Dean’s Nominee for societies',
+  FC: 'Fest Convenors',
+};
+
+const SectionNavbar1 = () => {
```
Comment (120EE0692) [inline client/src/components/widgets/SacInfoNavbar.js:24]: @anish-patro rename SectionNavbar1 avoid using numbers.

File: client/src/components/widgets/SocietyCards.js [hunk 8]
```diff
@@ -0,0 +1,132 @@
...
+          aria-label='show more'
+        >
+          <ExpandMore />
+        </IconButton>
+        <Typography variant='h3' className={classes.heading}>
+         {society}
+        </Typography>
+      </CardActions>
+      <Collapse in={expanded} timeout='auto' unmountOnExit>
+        <CardContent className={classes.hallDetails}>
```
Comment (120EE0692) [inline client/src/components/widgets/SocietyCards.js:47]: @anish-patro hallDetails ?

**Review by 120EE0692 — CHANGES_REQUESTED** [review]
@anish-patro code is not at all structured.

**Comment by 120EE0692** [comment]
@anish-patro please resolve all conversations, if required changes are done.

**Comment by rutajdash** [comment]
@120EE0692 @anish-patro what is up with this?

**Comment by 120EE0692** [comment]
> @120EE0692 @anish-patro what is up with this?

@rutajdash mostly done, only few club content is left. After that good to go.

File: client/src/pages/guide.jsx [hunk 9]
```diff
@@ -89,13 +89,13 @@ const GuidePage = ({ issues }) => {
...
-  };
-}
+// export async function getServerSideProps() {
+//   return {
+//     redirect: {
+//       destination: '/comingSoon',
+//       permanent: false,
+//     },
+//   };
+// }
```
Comment (120EE0692) [inline client/src/pages/guide.jsx:99]: @anish-patro guide page should be redirect to coming soon.

**Review by 120EE0692 — CHANGES_REQUESTED** [review]

**Review by 120EE0692 — APPROVED** [review]

**Review by rutajdash — APPROVED** [review]

**Pull request merged**